File: customview-study/src/main/java/com/asura/customview_study/application/MyApplication.java
package com.asura.customview_study.application;

import android.app.Application;

import com.asura.customview_study.utils.ALog;

/**
 * Created by Liuxd on 2016/9/14 14:22.
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        ALog.INSTANCE.initLog();
    }
}

File: jsoup-demo/src/main/java/com/asura/jsoup_demo/MyApplication.kt
package com.asura.jsoup_demo

import android.app.Application
import com.asura.jsoup_demo.util.ALog

/**
 * @author Created by Asura on 2018/6/7 14:34.
 */
class MyApplication : Application() {

    companion object {
        lateinit var INSTANCE: MyApplication;
        public fun getInstance(): MyApplication? {
            return INSTANCE;
        }
    }

    override fun onCreate() {
        super.onCreate()
        INSTANCE = this;
        ALog.initLog();
    }
}